is.js

```diff
@@ -169,7 +169,7 @@
     };
 
     // is a given value undefined?
-    is.undefined = function(value) {
+    is['undefined'] = function(value) {
         return value === void 0;
```

Comment:
IE8 doesn't have a problem with properties named `'undefined'`. I'd like to avoid superstitious code changes.